src/main/java/org/folio/service/pieces/flows/update/PieceUpdateFlowManager.java

```diff
@@ -55,11 +54,12 @@ public Future<Void> updatePiece(Piece pieceToUpdate, boolean createItem, boolean
     pieceStorageService.getPieceById(pieceToUpdate.getId(), requestContext)
       .onSuccess(holder::withPieceFromStorage)
       .compose(aHolder -> basePieceFlowHolderBuilder.updateHolderWithOrderInformation(holder, requestContext))
```

Comment:
Is order necessary in holder, because we get acq units from titles now?

Reply:
no, probably. we can't delete updating order information part because it is being used. 
![image](https://github.com/folio-org/mod-orders/assets/113523904/dfc175e9-2d50-4b0c-a7f3-ced20e94ddd4).

Reply:
isPieceRequestValid uses only po line, but not order. Maybe it is enought only to fetch order line but not order in updateHolderWithOrderInformation, for example by introducing updateHolderWithOrderLineInformation method?

Reply:
We Discussed this situation and decided to remain as it was